src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
+    public AuthenticationException(String message) {
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/200912630-877e9e82-8588-4a0b-af37-f2b549832f2c.png)

Reply:
not fixed